ArmoniK.Extension.CSharp.Worker/LibraryWorker.cs

```diff
@@ -128,4 +128,31 @@ public async Task<Output> ExecuteAsync(ITaskHandler      taskHandler,
              };
     }
   }
+
+  /// <summary>
+  ///   Checks the health of the library worker.
+  /// </summary>
+  /// <returns>True if the worker is healthy, false otherwise.</returns>
+  public bool CheckHealth(CancellationToken cancellationToken = default)
+  {
+    try
+    {
+      // Basic health checks
+      if (Configuration == null || LoggerFactory == null)
```

Comment:
This cannot be true. They are set in the constructor.

Also, why don't you check the health of the service inside your library worker?